oak/server/oak_runtime.cc

```diff
@@ -158,20 +156,6 @@ bool OakRuntime::CreateAndRunNode(const std::string& config_name,
 }
 
 grpc::Status OakRuntime::Start() {
-  // We call into the Rust runtime to verify that bindings between C++ and Rust are working
-  // correctly.
```

Comment:
Obviously this was just a placeholder, but I presume @blaxill's work on getting a live Rust runtime is going to clash with this?

Reply:
I'll let @blaxill comment on this. He told me it is safe to remove.... IIRC :)

Reply:
This is safe to remove. It was to check that we could compile no_std rust code into Asylo. Originally added by tzn, I continued to use it for asylo_rust code. The first iteration of rust oak_runtime I'm working on currently does not use asylo_rust but will in the future.